### PR TITLE
Packt URL added

### PR DESCRIPTION
**Pull request by packtpartner**
Added book

### CODE REVIEW

File: learn.md [hunk 1]
```diff
@@ -241,6 +241,15 @@ _Jamie Dixon_
 
 This book is packed with real-world examples to easily use machine learning techniques in your business applications.
 
+### [F# for Machine Learning Essentials](https://www.packtpub.com/big-data-and-business-intelligence/f-machine-learning)
+
+<img src="https://github.com/gavincardoza/fsfoundation/blob/gh-pages/about/files/F%23%20for%20Machine%20Learning%20Essentials.jpg" style="float:right;margin:5px 0px 5px 25px;" />
```
Comment (tpetricek) [inline learn.md]: The `img` path should be relative - as with the other books on the page.

File: about/index.md [hunk 2]
```diff
@@ -63,4 +63,17 @@ the standard F# library.
  * [Web Programming with F#](/guides/web)
 
  * [Data Access with F#](/guides/data-access)
+
```
Comment (tpetricek) [inline about/index.md]: This seems to be an unintended change - can you please revert changes in this file?

File: about/index.md [hunk 3]
```diff
@@ -63,4 +63,17 @@ the standard F# library.
  * [Web Programming with F#](/guides/web)
 
  * [Data Access with F#](/guides/data-access)
+ 
+ * Edit the link for this book [F# for Quantitative Finance] (https://www.packtpub.com/big-data-and-business-intelligence/f-quantitative-finance) to http://fsharp.org/learn.html
+ 
+ * Add this book [F# for Machine Learning Essentials] (https://www.packtpub.com/big-data-and-business-intelligence/f-machine-learning) to http://fsharp.org/learn.html
+
+  Please add the author's name along with the description for this book:
```
Comment (ReedCopsey) [inline about/index.md]: There are lines (like this one) that don't appear correct.  Please cleanup the descriptions and notes ("Edit the link for this book" and "Add this book" shouldn't be in the PR)

Comment (packtpartner) [inline about/index.md]: Hi Reed,
Should I only put only the book details? @ReedCopsey

Comment (ReedCopsey) [inline about/index.md]: Yes.  The text should match the surrounding examples.  We don't want the website to show text like "Please add the author's name along with the description for this book:" when published.

Comment (packtpartner) [inline about/index.md]: Thanks Reed.
I'll make changes and notify you once again.

Thanks once again

**Comment by packtpartner** [comment]
@dsyme can u help me with this?

**Comment by packtpartner** [comment]
@tpetricek -  Can you please look at this and let me know if any changes are to be made?

**Comment by tpetricek** [comment]
The change in `learn.md` looks okay, but please undo the changes in `about/index.md`.

**Comment by packtpartner** [comment]
@tpetricek the changes are done

File: about/index.md [hunk 4]
```diff
@@ -63,4 +63,15 @@ the standard F# library.
  * [Web Programming with F#](/guides/web)
 
  * [Data Access with F#](/guides/data-access)
+ 
+ * [F# for Quantitative Finance] (https://www.packtpub.com/big-data-and-business-intelligence/f-quantitative-finance) to http://fsharp.org/learn.html
```
Comment (ReedCopsey) [inline about/index.md]: These should be on the learn page, and not here in the guides list.  Please remove the additions to this file.

File: learn.md (hunk 1 above)
Comment (ReedCopsey) [inline learn.md]: This still needs changing.

**Comment by packtpartner** [comment]
I have created a new pull request. can u please have a look at it https://github.com/packtpartner/fsfoundation/pull/1

**Comment by tpetricek** [comment]
Closing this one.

**Pull request closed without merging**